src/Api/V1/Worker/ServiceInterface.php

```diff
@@ -37,6 +38,8 @@ public function getJobId(): int;
 
     public function getTimesRetried(): int;
 
+    public function getNewRelic(): Apm\NewRelicInterface;
```

Comment:
should this be `getApmNewRelic` to match the setter?

Reply:
The setter is the format of our auto-generated `AwareTrait`s, and includes the whole namespace (to prevent collisions), while the getter is what we actually want to expose to userspace

Reply:
We'll also be reworking the API in the SDK in `5.x`.